runtime/compilers/rillv1/parse_metrics_view.go

```diff
@@ -34,6 +34,7 @@ type MetricsViewYAML struct {
 		Name        string
 		Label       string
 		Column      string
+		Expression  string
```

Comment:
It looks like we don't have validation to ensure that only one of these are set (exactly one of them should be non-nil)